maquetado/index.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+
+<body>
+  <div class="textAndDescription">
```

Comment:
When it comes to just CSS & HTML we usually use components like classes, since they are more meaningful and you can avoid to crash into same name problems

With that in mind I would set this as `book-detail` instead